docs/overrides.md

```diff
@@ -46,15 +46,21 @@ It includes the same changes found in the previous file (devise/sessions/new.htm
 
 **decidim-ldap/app/views/layouts/decidim/_wrapper.html.erb**
 
+Override with Deface in **app/overrides/remove_signup_link_in_wrapper.rb**
+
 It modifies the lines 76 to 78 to remove the signup link for ldap enabled organizations.
 
 **app/views/decidim/authorization_modals/_content.html.erb**
 
+Override with Deface in **app/overrides/add_custom_error_messages_in_authorization_modals.rb**
+
 Starting at line 26, a new behaviour has been included to show custom error messages to the custom action authorizer created for
```

Comment:
Does this comment still apply? should we remove it?

Reply:
This comment is new, maybe we need rewrite it. 
Because the override is still done but with Deface. What do you think?

Reply:
I meant the comment at line 57, I think we can rewrite it, yes